src/cdk/drag-drop/drag-drop.md

```diff
@@ -1 +1,127 @@
-# TODO
+The `@angular/cdk/drag-drop` module provides you with a way to easily and declaratively create
+drag&drop interfaces, with support for free dragging, sorting within a list, transferring items
+between lists, animations, touch devices, custom drag handles, previews, and placeholders,
+in addition to horizontal lists and locking along an axis.
+
+### Getting started
+To get started, you have to include the `DragDropModule` in your app's imports and add the
+`cdkDrag` directive to the element that you want to be draggable. When a `cdkDrag` element is
+outside of a `cdk-drop` element, it's in "free dragging" mode, which means that the user will be
+able to drag it freely around the page.
+
+<!-- example(cdk-drag-drop-overview) -->
+
+### Sorting
+If you want to take your UI a step further and allow the user to re-arrange a list of data using
+drag&drop, you can do so using the `cdk-drop` directive. When a `cdk-drop` is added around some
+`cdkDrag` elements, they'll become grouped in a list and the CDK will automatically sort them
+as the user is dragging. Note that the CDK won't change the data for you, which means that you'll
+have to listen for the `dropped` event and re-arrange the date yourself once the user is done
+dragging. There are some provided utilities like the `moveItemInArray` and `transferArrayItem`
+functions that make it easier to manipulate your data.

```

Comment:
I see what you mean, but hopefully people will realize that they're just dealing with arrays. I added the functions since some things like move an item inside an array aren't trivial one-liners.